Sources/Application/Game/Word.swift

```diff
@@ -0,0 +1,20 @@
+public class Word {
+	public var gameWord: String
+
+	public init(gameWord: String) {
+		self.gameWord = gameWord
+	}
+
+	public func isValid() -> Bool {
+		return gameWord != "" && hasOnlyLetters()
+	}
+
+	private func hasOnlyLetters() -> Bool {
+		for character in gameWord.characters {
+			if !((character >= "a" && character <= "z") || (character >= "A" && character <= "Z")) {
+				return false
+			}
```

Comment:
I created a class `CharacterValidator` that adopts a `Validates` protocol. Both have method `hasOnlyLetters(subject: String)`. In my both my `Guess` and `Word` classes, I initialize `validator = CharacterValidator` and where I previously called `hasOnlyLetters` in those two classes, I now call `validator.hasOnlyLetters(gameWord or currentGuess)`